jenkins/cloud_resource_orchestration/Jenkinsfile

```diff
@@ -47,7 +47,7 @@ pipeline {
         }
         stage('Initial Cleanup') {
             steps {
-                 sh '''if [[ "$(podman images -q quay.io/athiru/cloud-governance 2> /dev/null)" != "" ]]; then podman rmi -f $(podman images -q quay.io/ebattat/cloud-governance 2> /dev/null); fi'''
+                 sh '''if [[ "$(podman images -q quay.io/athiru/cloud-governance 2> /dev/null)" != "" ]]; then podman rmi -f $(podman images -q quay.io/cloud-governance/cloud-governance 2> /dev/null); fi'''
```

Comment:
Pls create secret QUAY_CLOUD_GOVERNANCE_REPOSITORY instead of
quay.io/cloud-governance/cloud-governance